admin-portal/src/pages/packageDetails/packageDetails.tsx

```diff
@@ -92,6 +92,19 @@ export default function PackageDetails() {
       `Are you sure you want to change the status to ${statusToBeChanged}?`
     );
     if (confirmed) {
+      if (
+        aidPackageStatus &&
+        (statusToLevel[aidPackageStatus] === 1 ||
+          statusToLevel[aidPackageStatus] === 2) &&
```

Comment:
Use the enum,

aidPackageStatus == AidPackage.Status.Draft || ...

Reply:
Suggestion:
Keep an array of completed statuses (or incomplete statuses).
```js
const completedStatuses = [AidPackage.Status.Delivered, ....];
```


then we could check, 
```js
!completedStatuses.includes(aidPackageStatus) && completedStatuses.includes(statusToBeChanged)

```

So that we don't need to mess with numbers when adding a new status.

Reply:
Made the change, as requested in the new commit